examples/vision-camera/modules/webrtc-source/android/src/main/java/io/fishjam/example/webrtcsource/WebrtcVisionCameraCustomSource.kt

```diff
@@ -0,0 +1,19 @@
+package io.fishjam.example.webrtcsource
+
+import com.fishjamcloud.client.models.CustomSource
+import com.fishjamcloud.client.models.CustomSourceConsumer
+import com.fishjamcloud.client.models.VideoParameters
+import com.fishjamcloud.client.models.Metadata
+
+class WebrtcVisionCameraCustomSource: CustomSource {
+  override val isScreenShare = false
+  override val metadata: Metadata = mapOf("type" to "camera")
+  override val videoParameters = VideoParameters.presetFHD43
```

Comment:
Is it ok to hardcode it here? 🤔

Reply:
I thought that since it's just an example it's less code and passing around if I hardcode it.